testscommon/outportDataProviderStub.go

```diff
@@ -0,0 +1,37 @@
+package testscommon
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/data"
+	outportcore "github.com/ElrondNetwork/elrond-go-core/data/outport"
+)
+
+// OutportDataProviderStub -
+type OutportDataProviderStub struct {
+	PrepareOutportSaveBlockDataCalled func(
+		headerHash []byte,
```

Comment:
maybe PrepareOutportSaveBlock() can receive an Arg struct, instead of 5 parameters

Reply:
done